spec/services/zendesk_spec.rb

```diff
@@ -12,10 +12,14 @@
     def mock_zendesk_response(body)
       stub_request(
         :get,
-        /.*zendesk.com\/api\/v2\/.*/
-      ).with(
-        :body => "{\"query\":\"query:created>7days group:\\\"Incident response and tuning\\\" type:ticket\"}"
-      ).to_return(:status => 200, :body => body, :headers => {"Content-Type" => "application/json"})
+        /https:\/\/.*@ministryofjustice\.zendesk\.com\/api\/.*/
```

Comment:
Regarding the lack of V2 in the regex, I think its ok that if we do upgrade the version of the API that Zendesk API related tests fail.